site/content/docs/5.0/components/skeletons.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: docs
+title: Skeletons
+description: Create loading skeletons for your components or pages with Bootstrap skeleton, built entirely with HTML, CSS, and no JavaScript.
+group: components
+toc: true
+---
+
+## About
+
+Bootstrap “skeletons” can be used to enhance the experience of your application. They’re built only with HTML and CSS, meaning you don’t need any JavaScript to create them. You will, however, need some custom JavaScript to toggle their visibility. Their appearance, color, and sizing can be easily customized with our amazing utility classes.
```

Comment:
Please do not use curly double/single quotes. This is handled automatically for us.